pages/apply.html

```diff
@@ -42,18 +42,31 @@
 
         <div class="usa-input">
           <input type="checkbox" name="entry.1377828897" value="Verify that you meet the above eligibility requirements" id="group_1377828897_1" role="checkbox" class="ss-q-checkbox">
-          <label for="group_1377828897_1">Verify that you meet the above eligibility requirements</label>
+          <label for="group_1377828897_1">I, the applicant, verify that I meet the above eligibility requirements.</label>
+          <label for="group_1377828897_2">I, the applicant, verify that I do not meet the above eligibility requirements and will not be completing the rest of this application.</label>
         </div>
 
+    
+        
       </fieldset>
 
       <fieldset class="contact-information">
 
         <h2><legend>Contact Information:</legend></h2>
 
         <div class="usa-input name">
-          <label class="ss-q-item-label" for="entry_1741264393">Name <span class="required">*</span></label>
-          <input type="text" name="entry.1741264393" value="" class="ss-q-short" id="entry_1741264393" dir="auto" aria-label="Name  " aria-required="true" title="" required />
+                 
+          <p>
+          <label class="ss-q-item-label" for="entry_1741264393">First Middle Last Name <span class="required">*</span></label>
+          <br/>
+            If no middle name (NMN), write "NMN" between your first and last name. 
+          </p>
+            <input type="text" name="entry.1741264393" value="" class="ss-q-short" id="entry_1741264393" dir="auto" aria-label="First Middle Last Name  " aria-required="true" title="" required />
+        </div>
+        
+        <div class="usa-input preferredname">
+          <label class="ss-q-item-label" for="entry_732113689">Preferred Name <span class="required">*</span></label>
```

Comment:
i also added this and not sure if its "coded" correctly - and this is not optional.